src/main/resources/native/ios/AppDelegate.m

```diff
@@ -74,7 +71,23 @@ @implementation AppDelegate
 
 -(void)startVM:(id)selector {
     gvmlog(@"Starting vm...");
-    startGVM();
+    NSArray *documentPaths = NSSearchPathForDirectoriesInDomains(NSLibraryDirectory, NSUserDomainMask, YES);
+    NSString *documentsDir = [documentPaths objectAtIndex:0];
+    NSString *folder = @"gluon";
+    NSString *folderPath = [documentsDir stringByAppendingPathComponent:folder];
+    NSString *prop = @"-Duser.home=";
+    NSString *userhomeProp = [prop stringByAppendingString: folderPath];
```

Comment:
Probably no need to create new objects, just inline:
```
NSString *folderPath = [documentsDir stringByAppendingPathComponent:@"gluon"];
NSString *userhomeProp = [@"-Duser.home=" stringByAppendingString: folderPath];
```

Reply:
I prefer to keep those on separate lines. The resulting native code is the same, but I want to keep linenumber info as small as possible (if we see a crash in line X, I want a single statement to be the possible cause).